src/callout.tsx

```diff
@@ -16,6 +16,7 @@ import {sprinkles} from './sprinkles.css';
 import {vars} from './skins/skin-contract.css';
 import {getPrefixedDataAttributes} from './utils/dom';
 import {applyCssVars} from './utils/css';
+import {translate, closeButtonLabel as closeButtonLabelText} from './text-tokens';
```

Comment:
Current token names are too generic and cause collisions. For example: `clearButton` or `loading`. 
We would define a naming convention... For example `TextTokenLoading` or use uppercase names `TEXT_LOADING`

I'm open to suggestions

Reply:
I think you can do `import * as texts from './text-tokens'`
the bundler should be smart enough to detect `texts.something` usages

Reply:
It worked :D